ocs_ci/ocs/cluster.py

```diff
@@ -2635,6 +2636,63 @@ def exec_cmd_on_cluster_node(self, cmd):
         return_stdout = return_output[1]
         return return_stdout
 
+    def get_thin_provisioning_alerts(self):
+        """
+        Get the list of alerts that active in the cluster
+
+        Returns:
+            list: alrets name
+
+        """
+        new_prom = PrometheusAPI()
```

Comment:
Because we are getting the alerts couple of time we can have a function to initiate the prom and keep it.
Like:

def init_prom:
    self.new_prom = PrometheusAPI()

def get_thin_provisioning_alerts(self):
    if type(self.new_prom) is not PrometheusAPI:
        init_prom

Reply:
or initiate the PrometheusAPI instance as part of the `__init__()` method

Reply:
@ebenahar I don't think it is good. Prom initiation is long and we don't use it in every test.